config/hardware/accelerometers/adxl345_btt_lis2dw.cfg

```diff
@@ -0,0 +1,26 @@
+# This ADXL file is dedicated to be used with ADXL boards
+# connected over USB to the pi as dedicated and standalone ADXL-MCU boards
+
+# https://github.com/bigtreetech/LIS2DW
+
+# You need to override the following to be able to set the proper serial in your overrides.cfg file
+[mcu adxl]
+serial: /dev/serial/by-id/xxx
+
+[adxl345]
+cs_pin: adxl:gpio9
+#spi_bus: spi1a
+spi_software_sclk_pin: adxl:gpio10
+spi_software_mosi_pin: adxl:gpio11
+spi_software_miso_pin: adxl:gpio8
```

Comment:
But here is my problem... How to manage the pin name since it's not an ADXL chip anymore? I would suggest to use `accelerometer:pin_name` instead but this would need a change in all the ADXL files and is a potential breaking change if some users have overriden this in their config...